commerce_coordinator/apps/lms/views.py

```diff
@@ -74,16 +74,22 @@ def _redirect_response_payment(self, request):
         """
 
         get_items = list(self.request.GET.items())
-
-        redirect_url_obj = PaymentPageRedirectRequested.run_filter(request)
-
-        redirect_url = self._add_query_params_to_redirect_url(
-            redirect_url_obj['redirect_url'],
-            get_items
-        )
+        redirect_url = None
+        if "bundle" in dict(get_items):
+            ecom_url = urljoin(
+                settings.ECOMMERCE_URL, settings.ECOMMERCE_ADD_TO_BASKET_API_PATH
+            )
+            redirect_url = self._add_query_params_to_redirect_url(ecom_url, get_items)
+        else:
+            redirect_url_obj = PaymentPageRedirectRequested.run_filter(request)
+            redirect_url = self._add_query_params_to_redirect_url(
+                redirect_url_obj["redirect_url"], get_items
+            )
```

Comment:
Is there a reason not to add this new redirect logic to the pipeline steps? cc: @grmartin mind sharing your thoughts on this approach?

Reply:
The current pipeline steps are very specific to their own use case. Each pipeline step is descriptive of the work they are doing and I didn't want to add an additional responsibility in those steps.
I gave a thought to adding a new pipeline step as well but it seemed like an overkill and added complexity for a temporary measure that will be going away in near future.

Reply:
I think this is simple enough to pass for now. Ecomm is going away, itll be simpler to remove this way. Thanks for the check in though @shafqatfarhan